### PR TITLE
Release v2.5.0

### PR DESCRIPTION
**Pull request by mboersma**

### CODE REVIEW

File: workflow-v2.5.0/tpl/generate_params.toml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+
+[dockerbuilder]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.3.2"
+
+[controller]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.5.0"
```
Comment (bacongobbler) [inline workflow-v2.5.0/tpl/generate_params.toml]: can you bump this to v2.5.1 please?

File: workflow-v2.5.0/tpl/generate_params.toml [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+dockerTag = "v1.1.0"
+hostPort = 5555
+limits_cpu = "100m"
+limits_memory = "50Mi"
+
+[workflowManager]
+org = "deis"
+pullPolicy = "IfNotPresent"
+dockerTag = "v2.4.1"
+versionsApiURL = "https://versions-staging.deis.com"
```
Comment (kmala) [inline workflow-v2.5.0/tpl/generate_params.toml]: this and below should be changed to "https://versions.deis.com" and "https://doctor.deis.com"

Comment (mboersma) [reply workflow-v2.5.0/tpl/generate_params.toml]: I reverted them to the `-staging` URLs in a temp commit here, which is intended to be discarded before we actually merge this.

**Comment by vdice** [comment]
LGTM besides a friendly reminder in https://github.com/deis/charts/pull/348/files#r78082168 which looks like @mboersma has on top of mind!

**Pull request merged**